sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -462,7 +462,8 @@ def run_pipeline(self, pipeline, options):
     use_fnapi = apiclient._use_fnapi(options)
     from apache_beam.transforms import environments
     default_environment = environments.DockerEnvironment.from_container_image(
-        apiclient.get_container_image_from_options(options))
+        apiclient.get_container_image_from_options(options),
+        artifacts=environments.python_sdk_dependencies(options))
```

Comment:
Should we push populating artifacts into from_container_image?

Reply:
We need pipeline option to populate artifacts. I think we could either use `from_options` instead and override container image or just leave as is.

Reply:
Let's use from_options.